core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/project/ProjectResource.scala

```diff
@@ -348,6 +348,27 @@ class ProjectResource {
     }
   }
 
+  /**
+    * This method updates the description of a specified, existing project
+    *
+    * @param pid project ID
+    */
+  @POST
+  @Path("/{pid}/update/description")
+  @Consumes(Array(MediaType.APPLICATION_JSON))
+  @Produces(Array(MediaType.APPLICATION_JSON))
+  def updateProjectDescription(@PathParam("pid") pid: UInteger, project: UserProject): Unit = {
```

Comment:
why do we need a whole `Project` in the argument? I thought we could just have `pid, description` as the argument?

Reply:
Changed this to consume just a plain text string for the `description` rather than an entire `UserProject` JSON object, thanks!